index.bs

```diff
@@ -7245,6 +7284,33 @@ partial dictionary MLOpSupportLimits {
     1. Return |output|.
 </details>
 
+
+<div class="note">
+  The specific sampling algorithms are based on those widely used in existing Machine Learning frameworks. For example, when performing {{MLInterpolationMode/linear}} resampling from the following *[4, 4]* input tensor (considering only spatial dimensions):
```

Comment:
> The specific sampling algorithms are based on those widely used in existing Machine Learning frameworks.

> [!NOTE]
> Some ML libraries got this wrong historically and did things like align the top-left corner of the input to the top-left of the output, or stretch the *centers* of the input to the *corners* of the output, which graphics experts know is incorrect 😉 and gives you poor results. Imaging libraries like OpenCV though do the right thing, and thankfully newer versions of TF and PyTorch have fixed this behavior by default. e.g. [#1](https://hackernoon.com/how-tensorflows-tf-image-resize-stole-60-days-of-my-life-aba5eb093f35) [#2](https://discuss.pytorch.org/t/what-we-should-use-align-corners-false/22663/25).

(no action - resolve me)

Reply:
If we want to say more in the spec we can!

Reply:
Updated comment with visualization - think it would help? I should probably recreate it from scratch to avoid directly reusing Jacob Richeimer's figure https://jricheimer.github.io/tensorflow/2019/02/11/resize-confusion/.

Reply:
I'm thinking it will be hard to capture more of the history here without it turning into an essay equivalent to these linked resources. Maybe we should just link to these blog posts? @anssiko - any thoughts on non-normative links to potentially ephemeral resources?

Reply:
No need to stall on this aspect. Happy to merge if you say go.